src/main/scala/org/apache/spark/deploy/armada/Config.scala

```diff
@@ -157,6 +157,161 @@ private[spark] object Config {
   private def k8sLabelListValidator(labelList: String): Boolean = {
     parseCommaSeparatedK8sLabels(labelList).forall(_.isSuccess)
   }
+
+  val ARMADA_QUEUE: ConfigEntry[String] =
+    ConfigBuilder("spark.armada.queue")
+      .doc("The Armada queue that will be used for running the driver and executor pods.")
+      .version("1.0.0")
+      .stringConf
+      .createWithDefault("default")
+
+  val DRIVER_JOB_SET_ID: ConfigEntry[String] =
+    ConfigBuilder("spark.armada.driver.job.set.id")
+      .doc("The Armada job set id that will be used for running the driver pods.")
+      .version("1.0.0")
+      .stringConf
+      .createWithDefault("Driver-Job-Set")
+
+  val EXECUTOR_JOB_SET_ID: ConfigEntry[String] =
+    ConfigBuilder("spark.armada.executor.job.set.id")
```

Comment:
Should we use a single jobset for driver and executor?

Reply:
I think they should be part of the same jobset

Reply:
@ClifHouck is there a reason why are there two jobsets?

Reply:
I think it's fine to have one, there's no necessity for two afaik.

Reply:
There should be none, it should be derived from `spark.app.name`: https://github.com/armadaproject/armada-spark/pull/40#discussion_r2083053971